extensions/amp-share-tracking/0.1/test/test-amp-share-tracking.js

```diff
@@ -55,49 +57,59 @@ describe('amp-share-tracking', () => {
   }
 
   it('should get incoming fragment starting with dot', () => {
-    viewerGetFragmentMock.onFirstCall().returns(Promise.resolve('.12345'));
+    viewerGetFragmentStub.onFirstCall().returns(Promise.resolve('.12345'));
     return getAmpShareTracking().then(ampShareTracking => {
       return shareTrackingForOrNull(ampShareTracking.win).then(fragments => {
+        expect(viewerGetFragmentStub.callCount).to.equal(1);
+        expect(viewerUpdateFragmentStub.callCount).to.equal(1);
         expect(fragments.incomingFragment).to.equal('12345');
       });
     });
   });
 
   it('should get incoming fragment starting with dot and ignore ' +
       'other parameters', () => {
-    viewerGetFragmentMock.onFirstCall()
+    viewerGetFragmentStub.onFirstCall()
         .returns(Promise.resolve('.12345&key=value'));
     return getAmpShareTracking().then(ampShareTracking => {
       return shareTrackingForOrNull(ampShareTracking.win).then(fragments => {
+        expect(viewerGetFragmentStub.callCount).to.equal(1);
+        expect(viewerUpdateFragmentStub.callCount).to.equal(1);
```

Comment:
I see. Then let's verify the param passed to viewer as well.
expect(viewerGetFragmentStub.withArgs('.12345')).to.be.calledOnce;